README.md

```diff
@@ -516,10 +516,10 @@ the regular expression now it will find all matches in the input string, not jus
 
 The `m` modifier is used to perform a multi-line match. As we discussed earlier
 anchors `(^, $)` are used to check if pattern is the beginning of the input or
-end of the input string. But if we want that anchors works on each line we use
+end of the input string. But if we want that anchors work on each line we use
```

Comment:
:+1:

Reply:
Somewhat better:

`But if we want anchors to work on each line` (...)